RacingMod/Chat/ChatCommand.cs

```diff
@@ -0,0 +1,72 @@
+﻿using avaness.RacingMod.Race;
+using Sandbox.Game;
+using System;
+using System.Text;
+using VRage.Game.ModAPI;
+
+namespace avaness.RacingMod.Chat
+{
+    public abstract class ChatCommand
+    {
+        public abstract string Id { get; }
+        public abstract string Usage { get; }
+        public virtual string AdminUsage => Usage;
+        
+        public void Run(IMyPlayer p, bool admin, string[] cmd, Track race)
+        {
+            if (ValidateLength(cmd.Length, admin))
+            {
+                Execute(p, admin, cmd, race);
+            }
+            else
+            {
+                string usage = admin ? AdminUsage : Usage;
```

Comment:
- If the condition is true ShowChatMsg is called twice with essentially the same content.
- Inconsistent whitespace before {usage} between the two formatted string.

Reply:
Forgot the else statement.